dfastmi/io/RiverParameterData.py

```diff
@@ -136,13 +139,31 @@ def collect_int_values1(
             all_values.append(values_per_branch)
 
         return all_values
-
-
-    def collect_values_logical(
+    
+    def __read_value(self, key, branch, reach):
+        try:
```

Comment:
general_value = self._config.get("General", '').get(key, '')

No need for the try except blocks, maybe also more readable?

Reply:
or:
general = self._config.get("General", '')
general_value = general.get(key, '')

Reply:
unclear to me if this refactoring should be done in this issue or is inscope of`[DFAST-112](https://issuetracker.deltares.nl/browse/DFAST-112) Refactor RiverParameterData and related tests

Reply:
since the change was made in this issue I commented it in this issue, if you feel it is part of [DFAST-112](https://issuetracker.deltares.nl/browse/DFAST-112) you can implement it in that issue.